travis/jekyll_build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+if  [ -z ${JGT+x} ]; then
+    # We don't have access to the encrypted vars.
+    bundle exec jekyll build
+else:
+    # We do have access to the encrypted vars.
+    JEKYLL_GITHUB_TOKEN=$JGTbundle exec jekyll build
```

Comment:
There should be a space between $JGT and bundle.

Reply:
Fixed, thanks.